airflow/www/views.py

```diff
@@ -136,6 +136,11 @@
 PAGE_SIZE = conf.getint('webserver', 'page_size')
 FILTER_TAGS_COOKIE = 'tags_filter'
 FILTER_STATUS_COOKIE = 'dag_status_filter'
+LINECHART_X_AXIS_TICKFORMAT = (
+    "function (d, i) { let xLabel;"
+    "if (i === undefined) {xLabel = d3.time.format('%H:%M, %d %b %Y')(new Date(parseInt(d)));"
+    "} else {xLabel = d3.time.format('%H:%M')(new Date(parseInt(d)));} return xLabel;}"
```

Comment:
```suggestion
    "} else {xLabel = d3.time.format('%H:%M, %d %b')(new Date(parseInt(d)));} return xLabel;}"
```

I think we can get away with everything but the year. Showing only the hh:mm is really confusing if the chart is over the course of weeks or months.

Reply:
![image](https://user-images.githubusercontent.com/16971553/145312210-a9d0f207-4f51-4bda-8fb4-9e56076325b6.png)

Your suggestion looks pretty better than showing only the hh:mm 👍